src/test/java/com/jcabi/aspects/ImmutableTest.java

```diff
@@ -89,6 +91,35 @@ public void catchesTypesMutableByClassInheritance() {
     }
 
     /**
+     * Test Version.
+     */
+    @Test
+    public void testVersion() {
+        try {
+            new Mutable();
+        } catch (final IllegalStateException ex) {
+            MatcherAssert.assertThat(
+                ex.getMessage(),
+                Matchers.not(
+                    Matchers.containsString(
+                        "${projectVersion}"
+                    )
+                )
+            );
+            MatcherAssert.assertThat(
+                ex.getMessage(),
+                Matchers.not(
+                    Matchers.containsString(
+                        "${buildNumber}"
+                    )
+                )
+            );
+            return;
+        }
+        MatcherAssert.assertThat("should not happen.", false);
```

Comment:
@Suseika @pinaf 

OK. I still have to learn your rule (addressing comment). I just starts with this and one other issue.
Working on this. Will let you know when it is solved.